ngraph_bridge/ngraph_utils.h

```diff
@@ -27,13 +27,16 @@
 #include "tensorflow/core/platform/tensor_coding.h"
 #include "tensorflow/core/util/saved_tensor_slice_util.h"
 
-#include "ngraph/event_tracing.hpp"
 #include "ngraph/ngraph.hpp"
+#include "ngraph/runtime/chrome_trace.hpp"
 #include "ngraph/serializer.hpp"
 
 #include "logging/ngraph_log.h"
 #include "logging/tf_graph_writer.h"
 
+#define NG_TRACE(name, category, args) \
```

Comment:
How about we add a comment for the users of this MACRO?
// Activates event logging until the end of the current code-block scoping; Automatically writes log data as soon as the the scoping ends.